### PR TITLE
Add Bazel community roundup post for August 2022.

### PR DESCRIPTION
**Pull request by jin**

### CODE REVIEW

**Review by meisterT — APPROVED** [review]

File: _posts/2022-09-08-bazel-community-roundup.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+*   Slack is now kindly sponsoring https://slack.bazel.build, thanks to Erik Kerber.
+*   The frontend for bzlmod packages, [registry.bazel.build](http://registry.bazel.build), has a new UI [thanks to Max Goisser](https://twitter.com/Jakeherringbone/status/1559188586859180032).
+*   Bazel is an official project in the new [Google’s Open Source Software Vulnerability Rewards Program](https://security.googleblog.com/2023/08/Announcing-Googles-Open-Source-Software-Vulnerability-Rewards-Program%20.html).
+*   Radhika, PM for Bazel, is running a user study for Android App Development with Bazel. Please [fill up this form](https://forms.gle/XB2114bCrqSFNPHe9) if you develop Android apps.
+    *   [bazel-discuss thread](https://groups.google.com/g/bazel-discuss/c/298pjpxsciU)
+*   Bazel 6.0 LTS is now being prepared for release.
+    *   [Release tracker issue](https://github.com/bazelbuild/bazel/issues/16159)
+    *   [bazel-discuss thread](https://groups.google.com/g/bazel-discuss/c/oUB9-arSbYM)
+    *   Bazel 5.3rc2 is [available for testing](https://groups.google.com/g/bazel-discuss/c/512ou_Zu5LU).
```
[the remark this answers is not in the document]
Comment (jin) [reply _posts/2022-09-08-bazel-community-roundup.md]: Oops, yes.

File: _posts/2022-09-08-bazel-community-roundup.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Welcome to the August issue of the Bazel Community Round-up, where we collect and share
+content about Bazel over the month.
+
+**Announcements**
+
+*   Call for Proposals for BazelCon 2022 (Nov 16-17, New York City) is now open. [Submit your proposal here.](https://bit.ly/bazelcon2022cfp)
+    *   [BazelCon home page](https://conf.bazel.build/) 
+    *   [bazel-discuss thread](https://groups.google.com/g/bazel-discuss/c/QvNCK8z0ha8)
+*   Slack is now kindly sponsoring https://slack.bazel.build, thanks to Erik Kerber.
```
Comment (sventiffe) [inline _posts/2022-09-08-bazel-community-roundup.md:16]: Did you intentionally not linkify the slack link?

**Review by sventiffe — APPROVED** [review]

File: _posts/2022-09-08-bazel-community-roundup.md (hunk 2 above)
Comment (jin) [reply _posts/2022-09-08-bazel-community-roundup.md]: Not intentional - linkified.

**Review by jdob — APPROVED** [review]
LGTM

**Pull request merged**